rqd/rqd/rqmachine.py

```diff
@@ -552,17 +554,18 @@ def updateMachineStats(self):
                                          * mcpStat[statvfs.F_BSIZE]) / KILOBYTE
 
             # Reads dynamic information from /proc/meminfo
-            meminfoFile = open(rqconstants.PATH_MEMINFO, "r")
-            for line in meminfoFile:
+            with open(rqconstants.PATH_MEMINFO, "r") as fp:
+                meminfoLines = fp.readlines()
+            for line in meminfoLines:
```

Comment:
As a pattern I tend to prefer the following since it doesn't store the entire file in memory (probably not of concern with such a small file):
```
with open(rqconstants.PATH_MEMINFO) as fp:
    for line in fp:
        ...
```

Reply:
Good point, updated.